site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -178,102 +148,156 @@ files. To see if this is the problem, follow these steps:
+    - `/opt/google/chrome/extensions/`
+    - `/usr/share/google-chrome/extensions/`
+
+1. The following list describes extension installation from the Chrome Web Store, a CRX file or a
+   personal server:
+
+    - To install a **Chrome Web Store** extension, specify the update URL with the field name
+      "external_update_url". For example:
+      ```json
+      {
+        "external_update_url": "https://clients2.google.com/service/update2/crx"
+      }
+      ```
+    - To install the extension from a **CRX file**, specify the location in "external_crx" and the
+      version in "external_version". For example:
+      ```json
+      {
+        "external_crx": "/home/share/extension.crx",
+        "external_version": "1.0"
+      }
+      ```
+    - To install the extension **[hosted on a personal server][xml-update-url]**, the
+      "external_update_url" field has to point to the xml file, like in the following example:
+      ```json
+      {
+        "external_update_url": "http://myhost.com/mytestextension/updates.xml"
+      }
+      ```
+1.  Save the JSON file.
+1.  Launch Google Chrome and go to **chrome://extensions**; you should see the extension listed.
+
+{% Aside %}
+
+Use `chmod` if necessary to make sure that the `aaabbbcccdddeeefff.json` files are world-readable.
+Check the [preference file common mistakes FAQ][section-faq-preference] for additional help.
+
+{% endAside %}
+
+### Supported Locales {: #preference-locales }
+
+If you would like to install extension only for some browser locales, you can list supported locales
+in field name "supported_locales". Locale may specify parent locale like "en", in this case the
+extension will be installed for all English locales like "en-US", "en-GB", etc. If another browser
+locale is selected that is not supported by the extension, the external extensions will be
+uninstalled. If "supported_locales" list is missing, the extension will be installed for any locale.
+For example:
+
+```json
+{
+  "external_update_url": "https://clients2.google.com/service/update2/crx",
+  "supported_locales": [ "en", "fr", "de" ]
+}
+```
+
 ## Using the Windows registry {: #registry }
 
-1.  Find or create the following key in the registry:
-    - 32-bit Windows: `HKEY_LOCAL_MACHINE\Software\Google\Chrome\Extensions`
-    - 64-bit Windows: `HKEY_LOCAL_MACHINE\Software\Wow6432Node\Google\Chrome\Extensions`
-2.  Create a new key (folder) under the **Extensions** key with the same name as the ID of your
-    extension (for example, `aaaaaaaaaabbbbbbbbbbcccccccccc`).
-3.  In your extension key, create a property, "update_url", and set it to the value:
-    "https://clients2.google.com/service/update2/crx" (this points to your extension's crx in the
-    Chrome Web Store):
+1. Find or create the following key in the registry:
+
+    32-bit Windows
+
+    : `HKEY_LOCAL_MACHINE\Software\Google\Chrome\Extensions`
+
+    64-bit Windows
 
+    : `HKEY_LOCAL_MACHINE\Software\Wow6432Node\Google\Chrome\Extensions`
```

Comment:
@rdcronin, is these path still accurate?

Reply:
I *think* so

Reply:
I tested the 64-but path, it works